### PR TITLE
Adds support for aoe and aor

### LINKED ISSUE #2345: Implement `aor` command to show REIL of an instruction
`aoe` shuold display the esil . Currently it shows a TODO message which must be removed and replaced for this functionality. Help message must be updated

### PR DESCRIPTION
**Pull request by chinmaydd**
Aims to fix #2345.

### CODE REVIEW

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -303,12 +303,27 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 	RAnalOp op;
 	ut64 addr;
 	RAnalHint *hint;
+	// Variables required for setting up ESIL to REIL conversion
+	RAnalEsil *esil;
+	int	romem = r_config_get_i (core->config, "esil.romem");
+	int	stats = r_config_get_i (core->config, "esil.stats");
+	int	iotrap = r_config_get_i (core->config, "esil.iotrap");
+	int	stacksize = r_config_get_i (core->config, "esil.stacksize");
```
Comment (crowell) [inline libr/core/cmd_anal.c]: space instead of tab between `int` and the variable name.

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -334,6 +349,17 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 				free (d);
 			} else r_cons_printf ("Unknown opcode\n");
 			free (opname);
+		} else if (fmt == 'e') {
+			if (*R_STRBUF_SAFEGET (&op.esil)) {
+				r_cons_printf ("%s\n", R_STRBUF_SAFEGET (&op.esil));
+			}
+		} else if (fmt == 'r') {
+			if (*R_STRBUF_SAFEGET (&op.esil)) {
```
Comment (radare) [inline libr/core/cmd_anal.c]: Dont safeget twice. You a variable to cache the value

File: libr/core/cmd_anal.c [hunk 3]
```diff
@@ -661,7 +689,7 @@ static void r_core_anal_nofunclist  (RCore *core, const char *input) {
 				fcn = r_anal_get_fcn_in (core->anal, base_addr+chunk_offset, R_ANAL_FCN_TYPE_FCN | R_ANAL_FCN_TYPE_SYM);
 				if (fcn)
```
Comment (radare) [inline libr/core/cmd_anal.c]: Add braces {}

**Comment by radare** [comment]
There was already pie to print instructions esil, not sure if dupe at all here

**Comment by chinmaydd** [comment]
@radare true. `aoe` and `pie` basically satisfy the same functionality. Do you suggest we remove `aoe` and move `aor` into `pir`, maybe?

**Comment by radare** [comment]
Uhm dunno what others think. Maybe we can just make aoe an alias for pie with 1 as default

**Comment by radare** [comment]
Mainly to avoid dupped code and permite more than one way to do this. Looks consistent to me

**Comment by chinmaydd** [comment]
Cool! I will make the mentioned fixes soon.

File: libr/core/cmd_anal.c [hunk 4]
```diff
@@ -303,12 +303,13 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
...
 	ut64 addr;
 	RAnalHint *hint;
+	const char *esilstr;
 	// Variables required for setting up ESIL to REIL conversion
 	RAnalEsil *esil;
-	int	romem = r_config_get_i (core->config, "esil.romem");
-	int	stats = r_config_get_i (core->config, "esil.stats");
-	int	iotrap = r_config_get_i (core->config, "esil.iotrap");
-	int	stacksize = r_config_get_i (core->config, "esil.stacksize");
+	int romem = r_config_get_i (core->config, "esil.romem");
```
Comment (radare) [inline libr/core/cmd_anal.c]: this is bool

File: libr/core/cmd_anal.c [hunk 5]
```diff
@@ -303,12 +303,13 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
...
+	int stats = r_config_get_i (core->config, "esil.stats");
```
Comment (radare) [inline libr/core/cmd_anal.c]: this is bool too

File: libr/core/cmd_anal.c [hunk 6]
```diff
@@ -303,12 +303,13 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
...
+	int iotrap = r_config_get_i (core->config, "esil.iotrap");
```
Comment (radare) [inline libr/core/cmd_anal.c]: bool too

File: libr/core/cmd_anal.c [hunk 7]
```diff
@@ -303,12 +303,13 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
...
+	int stacksize = r_config_get_i (core->config, "esil.stacksize");
 	int use_color = core->print->flags & R_PRINT_FLAGS_COLOR;
```
Comment (radare) [inline libr/core/cmd_anal.c]: boolify this too

**Comment by radare** [comment]
fixed and merged. thanks!

**Pull request closed without merging**